src/common/plugin.ts

```diff
@@ -45,12 +47,27 @@ export abstract class Plugin implements IPlugin {
     const channelName = channel.name;
     const categoryName = channel.parent?.name.toLowerCase();
 
-    if (this.pluginChannelName && this.pluginChannelName !== channelName) {
-      const id = this.container.guildService.getChannel(this.pluginChannelName).id;
-      return `Please use this command in the <#${id}> channel.`;
+    const allowedChannels = [...(this.pluginChannelNames ?? [])];
+    if (this.pluginChannelName) {
+      allowedChannels.push(this.pluginChannelName);
     }
 
-    if (this.pluginCategoryName && this.pluginCategoryName.toLowerCase() !== categoryName) {
+    if (allowedChannels.length !== 0) {
+      if (!allowedChannels.includes(channelName)) {
+        const validChannelNames = allowedChannels
+          .map((name) => this.container.guildService.getChannel(name))
+          .join(', ');
+        return `Please use this command in ${
```

Comment:
I think you need to do the category check before denying the command because it’s not in the channel list

Reply:
Yep, realized this as soon as I pushed the last change(s).